jbpm-wb-process-runtime/jbpm-wb-process-runtime-api/src/main/java/org/jbpm/workbench/pr/model/DocumentSummary.java

```diff
@@ -24,6 +24,8 @@
 public class DocumentSummary extends GenericSummary<String> {
 
     private String documentId;
+    private String documentServerTemplateId;
+    private String documentIdentifier;
```

Comment:
yes, the documentId is the name+extension of the doc, for example myfile.properties
the documentIdentifier is the id that is used to retrieve the doc from DocumentServicesClient.